app/javascript/lib/dynamic_take_action_changes.js

```diff
@@ -1,19 +1,22 @@
-export function initTakeActionOnChangeHandlers() {
-    $("#hub_take_action_form_status, #hub_take_action_form_locale").change(function() {
+export function initTakeActionOnChangeHandlers(formName) {
+
+    $(`#hub_${formName}_form_status, #hub_${formName}_form_locale`).change(function() {
         $(".hub-form :input").prop("disabled", true);
-        const pathname = window.location.pathname.replace("update_take_action", "edit_take_action");
-        window.location.replace(pathname + "?" + searchParamsString() + "#status");
+        const pathname = window.location.pathname.replace(`update`, `edit`);
```

Comment:
can change this to normal quotes if we want to now since no string interpolation is happening.